examples/cloud-tomee-azure/README_pt.adoc

```diff
@@ -0,0 +1,111 @@
+= Implantação de TomEE no Azure
+:index-group: Cloud
+:jbake-type: page
+:jbake-status: published
+
+Este exemplo usa um aplicativo de eco básico, implantado com o TomEE incorporado na nuvem do Azure.
+
+Usamos o plugin do maven do TomEE para empacotar o aplicativo com o TomEE Embedded
+a fim de gerar um fat jar. Este jar é então escolhido e deployado pelo azure-webapp-maven-plugin.
+
+== Configuração do Azure
+
+Para que o plug-in do Azure funcione, você precisará ter uma conta do Azure e adicionar uma assinatura a ela.
+
+Então, em sua máquina de desenvolvimento, instale a interface da linha de comandos (CLI) do Azure e faça a autenticação com a linha de comando, 
+antes que você possa implantar seu aplicativo.
+
+- Crie uma conta Azure, se você não tem uma, faça aqui https://azure.microsoft.com/en-us
+- Use a opção free, se disponível ou https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade[adicionar uma assinatura].
+- https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Instalar] o Azure (CLI) de acordo com o sistema operacional do computador que você está usando para desenvolver.
+
+- Finalmente, você pode configurar seu computador de desenvolvimento.
+
+=== Fazendo login no Azure
+
+`az login`
+
+O resultado:
+
+----
+[
+   {
+     "cloudName": "AzureCloud",
+     "id": "aaaaaaaa-aaaa-aaaa-aaaaa-aaaaaaaaaa",
+     "isDefault": true,
+     "name": "Pay-As-You-Go",
+     "state": "Enabled",
+     "tenantId": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb",
+     "user": {
+       "name": "<o email da sua conta azure>",
+       "type": "user"
+     }
+   }
+ ]
+----
+O TenantId é alguém que pode registrar e gerenciar aplicativos por conta própria. Você precisará disso para mais tarde.
+
+
+=== Criando um service principal
+Um Azure service principal é uma identidade de segurança usada por aplicativos, serviços e
+e ferramentas de automação para acessar recursos específicos do Azure:
+
+----
+az ad sp create-for-rbac --name  http://<seu-sub-domínio> --password <password para este aplicativo>
+
+{
+  "appId": "cccccccc-cccc-cccc-cccc-ccccccccccccccc",
+  "displayName": "cloud-tomee-azure",
+  "name": "http://cloud-tomee-azure",
+  "password": "<password for this app>",
+  "tenant": "bbbbbbb-bbbbb-bbbb-bbbbb-bbbbbbbbbbb"
+}
+----
+O <seu-sub-domínio>, também chamado de service principal na documentação do Azure.
```

Comment:
In english has "also called the service principal name" so in portuguese I think is better "também é chamado de service principal name (nome principal do serviço)" because on azure doc is referenced by "service principal name".

Reply:
done. thanks.